s3-private/variables.tf

```diff
@@ -84,3 +84,8 @@ variable "cors_expose_headers" {
   type    = list(string)
   default = ["ETag"]
 }
+
+variable "sse_algorithm" {
+  type    = string
+  default = "AES256"
```

Comment:
Can we validate, that the value is either `AES256` or `aws:kms`? The validation is not only a validation, but also documentation for us :)